src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +307,77 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = `${this.aircraftModel.callsign}`;
```

Comment:
I don't understand this line? `callsign` should already be a string, right?

Reply:
mmm, yeah.  that's a good catch.  I'll update that.